demos/interactive_face_detection_demo/main.cpp

```diff
@@ -169,6 +170,9 @@ int main(int argc, char *argv[]) {
         // --------------------------- 3. Doing inference -----------------------------------------------------
         // Starting inference & calculating performance
         slog::info << "Start inference " << slog::endl;
+        if (!FLAGS_no_show) {
+            std::cout << "Press any key to stop" << std::endl;
+        }
```

Comment:
This block was intentionally removed by me because there is 
```c++
        std::cout << "To close the application, press 'CTRL+C' here";
        if (!FLAGS_no_show) {
            std::cout << " or switch to the output window and press any key";
        }
````
below. Please, revert.

Reply:
Agree. Done.